tests/fuzz/wasm-mutator-fuzz/server/app/main.py

```diff
@@ -204,7 +204,7 @@ def New_fuzzing():
         # curd.set_error_status_to(list(map(lambda x: x.id, error_list)), db)
         # Fuzzing.query.filter_by(id=fuzz.id).delete()
         fuzz.data = {'error': "Clone repo Error"}
-        db.commit()
+        db.session.commit()
```

Comment:
is this a functional change?

Reply:
This line should be a typo, the previous lines(such as L194) all use `db.session.commit()`